### ISSUE TITLE
brew install failing

### ISSUE THREAD

**Issue by korroktheslavemaster** [influxdata/influxdb#10412]
__System info:__ [Mac OS El Capitan 10.11.6]

__Steps to reproduce:__

1. `brew install influxdb`

gives:
```
Error: An exception occurred within a child process:
  RuntimeError: v1.6.4 tag should be c75cdfdfa6f71a08473fefcec71f6cbcbdef1ff4
but is actually a2ba6e7654fb078f8a3f5add1f8d935df38161bd
```

**Comment by pavankumar2203** [comment]
+1

**Comment by pavankumar2203** [comment]
upgraded Mac OS to Sierra and it worked

**Comment by stevebang** [comment]
Inadvertently, we caused a new commit for the InfluxDB 1.6.4 OSS release. As a result, the commit ID changed and was updated in the Homebrew formula for influxdb.  
- If the previous 1.6.4 formula was installed, brew will not attempt to update it. The files are identical, but the commit is different -- `brew upgrade` only checks the version for influxdb, which is still 1.6.4.
- I uninstalled influxdb (`brew uninstall influxdb`) and then installed it using `brew install influxdb` and 1.6.4 with the newer commit was installed.
